nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -434,13 +429,18 @@ let
     auth_basic secured;
     auth_basic_user_file ${auth_file};
   '');
+
   mkHtpasswd = name: authDef: pkgs.writeText "${name}.htpasswd" (
     concatStringsSep "\n" (mapAttrsToList (user: password: ''
       ${user}:{PLAIN}${password}
     '') authDef)
   );
 
   mkCertOwnershipAssertion = import ../../../security/acme/mk-cert-ownership-assertion.nix;
+
+  mkProxyCache = concatStringsSep "\n" (mapAttrsToList (name: proxyCache: ''
+    proxy_cache_path /var/cache/nginx/${name} keys_zone=${proxyCache.keysZoneName}:${proxyCache.keysZoneSize} levels=${proxyCache.levels} use_temp_path=${if proxyCache.useTempPath then "on" else "off"} inactive=${proxyCache.inactive} max_size=${proxyCache.maxSize};
```

Comment:
Can you format the parameter on multiple lines so that they are readable?
Also, please format the function calls (`filterAttrs` `mapAttrsToList`, etc.) so we can read them.